fetch.bs

```diff
@@ -7224,22 +7242,27 @@ of running <a for=Body>consume body</a> with <a>this</a> and the following step
 <p>The <dfn method for=Body><code>blob()</code></dfn> method steps are to return the result
 of running <a for=Body>consume body</a> with <a>this</a> and the following step given a
 <a for=/>byte sequence</a> <var>bytes</var>: return a {{Blob}} whose contents are <var>bytes</var>
-and whose {{Blob/type}} attribute is <a>this</a>'s <a for=Body>MIME type</a>.
+and whose {{Blob/type}} attribute is to <a for=Body>get the MIME type</a> with <a>this</a>.
 </div>
 
 <div algorithm>
 <p>The <dfn method for=Body><code>formData()</code></dfn> method steps are to return the result of
 running <a for=Body>consume body</a> with <a>this</a> and the following step given a
-<a for=/>byte sequence</a> <var>bytes</var>: switch on <a>this</a>'s <a for=Body>MIME type</a>'s
-<a for="MIME type">essence</a> and run the corresponding steps:
+<a for=/>byte sequence</a> <var>bytes</var>:
+
+<ol>
+<li><p>Let <var>mimeType</var> be the result of <a for=Body>get the MIME type</a> with <a>this</a>.
+<li>
+  <p>Switch on <var>mimeType</var>'s <a for="MIME type">essence</a> and run the corresponding
+  steps:
```

Comment:
It looks like the text here was wrong before as MIME type can be null. So we probably need to pull the "Otherwise" branch forward and deal with that here.

I guess it's fair if you want to leave that as a follow-up as you didn't introduce it.

Reply:
As this patch aims to clarify around MIME type here, it sounds good to me to handle this along with this PR! Thanks for catching it!

> 1. If _mimeType_ is null, [Throw](https://webidl.spec.whatwg.org/#dfn-throw) a [TypeError](https://webidl.spec.whatwg.org/#exceptiondef-typeerror).
> 2. Otherwise, switch on [....]

Does this ☝️ sound good to you?